Sources/Sentry/Public/SentryEvent.h

```diff
@@ -17,9 +17,9 @@ NS_SWIFT_NAME(Event)
 @property (nonatomic, strong) SentryId *eventId;
 
 /**
- * Message of the event
+ * Message of the event.
  */
-@property (nonatomic, strong) SentryMessage *message;
+@property (nonatomic, strong) SentryMessage *_Nullable message;
```

Comment:
L: why not using the `nullable`? guess Ive seen you suggesting this over the other on another PRs